addons/stock/data/stock_data.xml

```diff
@@ -118,9 +118,14 @@
         <function model="res.company" name="create_missing_scrap_location"/>
         <function model="res.company" name="create_missing_scrap_sequence"/>
 
+        <!-- Config Parameters -->
         <record model="ir.config_parameter" id="barcode_separator" forcecreate="False">
             <field name="key">stock.barcode_separator</field>
             <field name="value">,</field>
         </record>
+        <record model="ir.config_parameter" id="visibility_days" forcecreate="True">
```

Comment:
```suggestion
        <record model="ir.config_parameter" id="visibility_days">
```
Nitpick: [Documentation](https://www.odoo.com/documentation/master/developer/reference/backend/data.html#record) says that `forcecreate` defaults to `True` if not specified.